ssl_cert_tracker/models.py

```diff
@@ -13,63 +13,93 @@
 :contact:    ali.rahmat@phsa.ca
 
 """
+from django.conf import settings
 from django.db import models
-from django.db.models import signals
-from django.dispatch import receiver
+from p_soc_auto_base.models import BaseModel
 
-class TestCertsData(models.Model):
-    """TestCertsData. Model struture for NMap result response. """
-    orion_id =  models.CharField(max_length=100, blank=True, null=True)
-    addresses =  models.CharField(max_length=100, blank=True, null=True)
+class NmapCertsData(BaseModel, models.Model):
+    """NmapCertsData. Model struture for NMap result response. """
+    orion_id = models.CharField(max_length=100, blank=True, null=True)
+    addresses = models.CharField(max_length=100, blank=True, null=True)
     not_before = models.DateTimeField(null=True, blank=True)
     not_after = models.DateTimeField(null=True, blank=True)
     xml_data = models.TextField()
-    common_name =  models.CharField(max_length=100, blank=True, null=True)
-    organization_name =  models.CharField(max_length=100, blank=True, null=True)
-    country_name =  models.CharField(max_length=100, blank=True, null=True)
-    sig_algo =  models.CharField(max_length=100, blank=True, null=True)
-    name =  models.CharField(max_length=100, blank=True, null=True)
-    bits =  models.CharField(max_length=100, blank=True, null=True)
-    md5 =  models.CharField(max_length=100, blank=True, null=True)
-    sha1 =  models.CharField(max_length=100, blank=True, null=True)
-    
-    xml_data =  models.TextField(blank=True, null=True)
+    common_name = models.CharField(max_length=100, blank=True, null=True)
+    organization_name = models.CharField(max_length=100, blank=True, null=True)
```

Comment:
please add help_text properties to fields mentioning the relevant RFC field mappings